x-pack/plugins/maps/public/connected_components/layer_addpanel/source_editor/view.js

```diff
@@ -1,40 +0,0 @@
-/*
```

Comment:
Inlined component in `AddLayerPanel`. After removing duplicated button, the component is really small and does not need its own definition.